src/TicTacToeLoop.py

```diff
@@ -0,0 +1,48 @@
+import sys 
+import numpy as np
+
+from ComputerTurn import getComputerInput
+from UserTurn import getUserMove
+from WinningCombo import checkForWin
+
+#  Game runner Responsibilities
+def checkForCatsGame(inputArray):
+    if len(inputArray) == 0:
+        return True
+
+    else:
+        return False
+
+def printBoard(gameBoard):
+    print("\n %s | %s | %s \n---+---+---\n %s | %s | %s \n---+---+---\n %s | %s | %s \n" % \
+    (gameBoard[0][0], gameBoard[0][1], gameBoard[0][2],
+    gameBoard[1][0], gameBoard[1][1], gameBoard[1][2],
+    gameBoard[2][0], gameBoard[2][1], gameBoard[2][2]))
+
+def oneTurn(gameBoard, inputArray):
+    printBoard(gameBoard)
+    
+    gameBoard, inputArray = getUserMove(gameBoard, inputArray)
+
+    if checkForCatsGame(inputArray) == True:
+        return gameBoard, inputArray
+
+    if checkForWin(gameBoard) == True:
+        return gameBoard, inputArray
+
+    gameBoard, inputArray = getComputerInput(gameBoard)
+
+    return gameBoard, inputArray
+
+def takeTurns(board, possibleInputs):
+
+    while checkForWin(board) == False and checkForCatsGame(possibleInputs) ==False:
+        board, possibleInputs = oneTurn(board, possibleInputs)
```

Comment:
I like the way this while loop reads. While nobody's won and it's not a cat game, take a turn.